src/github.com/getlantern/flashlight/config/fetcher.go

```diff
@@ -117,8 +118,9 @@ func (cf *fetcher) fetchCloudConfig(cfg *Config) ([]byte, error) {
 	req.Header.Set("Lantern-Fronted-URL", cfg.FrontedCloudConfig+cb)
 
 	id := cf.user.GetUserID()
-	if id != "" {
-		req.Header.Set(userIDHeader, id)
+	if id != 0 {
+		strId := strconv.FormatFloat(id, 'f', -1, 64)
+		req.Header.Set(userIDHeader, strId)
```

Comment:
Right -- that would only happen if actually sent 12.14 somewhere as the user ID, which would cause all sorts of other problems @xiam.